src/utils/accessibility/accessibility.js

```diff
@@ -3,20 +3,32 @@ function isLargeScale(fontSize, isFontBold) {
   return isFontBold ? (fontSize >= 14) : (fontSize >= 18);
 }
 
-function accessibleContrast(accessibilityLevel, fontSize, isFontBold) {
+function fontSizeInPt(fontSize,fontUnitOfMeasure) {
+  // http://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html#visual-audio-contrast-contrast-73-head
+  return (fontUnitOfMeasure === 'pt') ? fontSize : Math.round(fontSize * 0.75);
+}
+
+function fontSizeInPx(fontSize,fontUnitOfMeasure) {
+  // http://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html#visual-audio-contrast-contrast-73-head
+  return (fontUnitOfMeasure === 'pt') ? Math.round(fontSize * 1.333) : fontSize;
```

Comment:
How about we put this magical number in a constant at the top of this file:

```js
// 1pt = 1.333px according to http://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html#visual-audio-contrast-contrast-73-head
const PIXELS_IN_POINT = 1.333;
```

To use it, we'll do `fontSize * PIXELS_IN_POINT` or `fontSize / PIXELS_IN_POINT`.